test/ecto/schema_test.exs

```diff
@@ -69,6 +69,10 @@ defmodule Ecto.SchemaTest do
     assert Ecto.Model.primary_key(%SchemaModel{uuid: "hello"}) == "hello"
   end
 
+  test "has __state__ attribute" do
```

Comment:
Can we add an assertion that checks that `SchemaModel.__schema__(:__state__) == nil`?